torch/onnx/utils.py

```diff
@@ -773,11 +772,11 @@ def _run_symbolic_function(g, n, inputs, env, operator_export_type=OperatorExpor
                     return g.op("Constant", value_t=n["value"])
                 if n.kindOf("value") == "s":
                     return g.op("Constant", value_s=n["value"])
-                elif n.kindOf("value") == "is":
-                    value = torch.stack([torch.tensor(v) for v in n["value"]]) if n["value"] else []
+                elif n.output().type().isSubtypeOf(ListType.ofInts()):
+                    value = torch.stack([torch.tensor(v) for v in n.output().toIValue()])
```

Comment:
empty list not valid input for `torch.stack`
```suggestion
                    vals = n.output().toIValue()
                    value = torch.stack([torch.tensor(v) for v in vals]) if len(vals) > 0 else []
```